root/src/main/java/com/example/servingwebcontent/movieObject.java

```diff
@@ -0,0 +1,21 @@
+package com.example.servingwebcontent;
+
+import java.util.*;
+
+public class MovieObject {
```

Comment:
You don't need "Object" in "MovieObject", it's a bit redundant since all classes in Java inherit from the Object class.